packages/wallet-sdk/src/store/config.ts

```diff
@@ -1,34 +0,0 @@
-import { createJSONStorage, persist } from 'zustand/middleware';
-import { createStore } from 'zustand/vanilla';
-
-import { VERSION } from '../sdk-info.js';
-import { AppMetadata, Preference } from ':core/provider/interface.js';
-
-export type ConfigState = {
-  metadata?: AppMetadata;
-  preference?: Preference;
-  version: string;
-};
-
-export const sdkconfig = createStore(
-  persist<ConfigState>(
-    () => ({
-      metadata: undefined,
-      preference: undefined,
-      version: VERSION,
-    }),
-    {
-      name: 'cbwsdk.config',
```

Comment:
does moving the config to different key break existing stored configs?

Reply:
No this value might persist but no versions of the SDK have been published since this config store was added